doc/classes/Vector2i.xml

```diff
@@ -174,7 +174,7 @@
 			</description>
 		</operator>
 		<operator name="operator *">
-			<return type="Vector2i" />
+			<return type="Vector2" />
```

Comment:
The documentation for the operators should be updated too as it documented the bugged behaviour.

Reply:
@reduz You also need to update the code examples.